keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/BakeryLifecycleMonitor.kt

```diff
@@ -123,7 +123,8 @@ class BakeryLifecycleMonitor(
       status = status,
       link = orcaTaskIdToLink(this),
       text = text,
-      startMonitoring = false
+      startMonitoring = false,
+      timestamp = null // let repository record the current time
```

Comment:
Not sure I understand the practical difference here

Reply:
Otherwise we were copying forward the start time of the event. This lets the timestamp be calculated when the event is saved into the database